lib/odbc.js

```diff
@@ -1196,28 +1193,28 @@ Pool.prototype.open = function (connStr, callback)
   //check to see if we already have a connection for this connection string
   if (self.availablePool[connStr] && self.availablePool[connStr].length)
   {
-    db = self.availablePool[connStr].shift()
+    db = self.availablePool[connStr].shift();
     db.lastUsed=null;
-    self.usedPool[connStr].push(db)
+    self.usedPool[connStr].push(db);
     callback(null, db);
   }
-  else if((self.maxPoolSize > 0) && (self.poolSize == self.maxPoolSize))
+  else if((self.maxPoolSize > 0) && (self.poolSize === self.maxPoolSize))
```

Comment:
@gildean This change should be to <== however I believe @bimaljha will be merging in this single change in a different PR.  May need to check for <= vs <== in that change.